purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -144,6 +145,13 @@ class Purchases internal constructor(
     val store: Store
         get() = purchasesOrchestrator.store
 
+    /**
+     * The currently configured FileRepository
+     */
+    @OptIn(InternalRevenueCatAPI::class)
```

Comment:
This should propagate the experimental label to consumers like this:
```suggestion
    @InternalRevenueCatAPI
```

This way, consumers need to opt in to access this

Reply:
@JZDesign we should do this before making a release, so we don't create a new stable API and we're forced to make a breaking change later :)

Reply:
https://github.com/RevenueCat/purchases-android/pull/2709